tests/processes/BearRunningTest.py

```diff
@@ -253,13 +253,16 @@ def setUp(self):
         self.file_name_queue.put("invalid file")
         self.local_bear_list.append(LocalTestBear(self.settings,
                                                   self.message_queue))
-        self.local_bear_list.append("not a valid bear")
+        self.local_bear_list.append(SimpleGlobalBear({},
+                                                     self.settings,
+                                                     self.message_queue))
```

Comment:
sorry if I haven't responded to some questions, with the github notification system it's very hard to determine where my attention is needed.